src/Handler/StreamHandler.php

```diff
@@ -61,9 +61,9 @@ public function __invoke(RequestInterface $request, array $options): PromiseInte
             $message = $e->getMessage();
             // This list can probably get more comprehensive.
             if (\strpos($message, 'getaddrinfo') // DNS lookup failed
```

Comment:
Shouldn't we add the `!== false` to the line here?

Reply:
fixed in 5326569762e076fc8e5580277fcda355ee13a68b